CMakeLists.txt

```diff
@@ -131,7 +131,9 @@ else (APPLE)
 endif(APPLE)
 
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+if (NOT CMAKE_SKIP_RPATH) # Option to not set rpath - required for packaging on Debian
```

Comment:
I tried without the IF and it works as documented. I also found I could remove the IF on [line 411 (was line 414)](https://github.com/shoogle/MuseScore/blob/debian-packaging/CMakeLists.txt#L411) by setting the variable CMAKE_INSTALL_RPATH instead of manually setting the compiler flags.